src/k8s/pkg/k8sd/api/cluster.go

```diff
@@ -50,5 +50,8 @@ func (e *Endpoints) getClusterStatus(s *state.State, r *http.Request) response.R
 		},
 	}
 
+	// Hide metrics-server in the cluster status response
+	result.ClusterStatus.Config.MetricsServer.Enabled = nil
+
```

Comment:
let's revert this line, and instead ignore it on the client side:

In https://github.com/canonical/k8s-snap/blob/ffb494961acf113a2af715a35e634c8b6a522a53/src/k8s/cmd/k8s/k8s_get.go#L29-L36

```go
			// ignore MetricsServer config
			config.MetricsServer = apiv1.MetricsServerConfig{}
```

- In https://github.com/canonical/k8s-snap/blob/ffb494961acf113a2af715a35e634c8b6a522a53/src/k8s/cmd/k8s/k8s_status.go#L31-L37

```go

			// ignore MetricsServer config
			status.Config.MetricsServer = apiv1.MetricsServerConfig{}
```